resources/classdetail/disk/compute_disk.go

```diff
@@ -0,0 +1,117 @@
+package disk
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+type diskJSON struct {
+	Name              string
+	Region            string
+	Zone              string
+	DefaultDiskSizeGb string
+	ValidDiskSize     string
+}
+
+type disk struct {
+	Type           string
+	Region         string
+	Zone           string
+	DefaultSizeGiB int64
+	MinSize        int64
+	MaxSize        int64
+}
+
+// Disks are stored first by type, then by zone/region.
```

Comment:
Nit: The comment is helpful, but it confused me a bit, too. Can you make it clearer whether it's about disk types or concrete disks? I should be about disk types, IIUIC, otherwise it should not be a package variable. Maybe you can add a note to the comment explaining where/when this variable is initialized.